cmd/insolar/main.go

```diff
@@ -20,6 +20,7 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"github.com/insolar/insolar/api"
```

Comment:
please setup your dev environment accordingly to wiki

Reply:
okey